### PR TITLE
Removed sha1 data and correct date format

### LINKED ISSUE #3182: Remove SHA1 field from Registry key detail
Remove SHA1 field from Registry key detail

**Sub-task:**
Also, check for old date formats in the section.

**Optional**
Improve some unit tests.

### PR DESCRIPTION
**Pull request by sortizowlh**
Hi team!
This PR removes SHA1 data from windows registry tab
Also, applies date format to every windows table with date
Closes #3182

### CODE REVIEW

**Review by gabiwassan — APPROVED** [review]
LGTM!

File: public/components/agents/fim/inventory/fileDetail.tsx [hunk 1]
```diff
@@ -260,7 +255,7 @@ export class FileDetails extends Component {
         'YYYY-MM-DD'
       )} AND ${field}<${value_max.format('YYYY-MM-DD')}`;
     } else {
-      newBadge.value = `${field}=${field === 'size' ? this.props.currentFile[field] : value}`;
+      newBadge.value = `${field}=${field === 'size' ? this.props.currentFile[field] : value}`;
```
Comment (Desvelao) [inline public/components/agents/fim/inventory/fileDetail.tsx:258]: Could remove the extra spaces at the end of the line?

File: public/components/agents/fim/inventory/filterBar.tsx [hunk 2]
```diff
@@ -55,7 +53,7 @@ export class FilterBar extends Component {
   }
 
   render() {
-    const { onFiltersChange, selectView, filters} = this.props;
+    const { onFiltersChange, selectView, filters} = this.props;
```
Comment (Desvelao) [inline public/components/agents/fim/inventory/filterBar.tsx:56]: Could remove the extra spaces at the end of the line?

**Review by frankeros — APPROVED** [review]
LGTM!

**Pull request merged**